boards/shields/mikroe_wifi_bt_click/boards/lpcxpresso55s69_cpu0.overlay

```diff
@@ -0,0 +1,7 @@
+/* Copyright (c) 2022 Linaro Limited.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&mikrobus_serial {
+	status = "okay";
+};
```

Comment:
I think this shouldn't be required, as this is part of mikroe_wifi_bt_click.overlay